### ISSUE TITLE
Should `keys-pub/tap/libfido2` redirect to `libfido2`?

### ISSUE THREAD

**Issue by tylerd-canva** [keys-pub/homebrew-tap#4]
This URL 404s: https://github.com/keys-pub/homebrew-tap/blob/master/libfido2.rb#L4

And the `keys-pub/go-libfido2` README was recently updated to point away from it: https://github.com/keys-pub/go-libfido2/commit/4467c2168a57dc5a9d49ef1160290fd6376241cd

**Comment by gabriel** [comment]
Yeah the patches got applied awhile back and no need to keep a seperate tap.

I probably should have kept the old one around but... also probably good for people to be using more recent libfido as well so...
